### ISSUE TITLE
How set Locale Flatpickr and full calendar

### ISSUE THREAD

**Issue by gianlucafiore** [saltcorn/saltcorn#1413]
**how can I do to set Spanish language in flatpickr and full calendar??**

![image](https://user-images.githubusercontent.com/50465150/194801534-da6bd85d-520f-47d9-b3c1-213ba46bfa0a.png)

**Comment by gianlucafiore** [comment]
@glutamate I generated a pull request to solve this, I hope it will be useful and you can perform the merge
best regards!

**Comment by kevin8181** [comment]
fullcalendar already uses whatever locale is set in your browser

**Comment by kevin8181** [comment]
no, fullcalendar is a separate plugin for displaying events from the
database on a calendar

**Comment by skaskiewicz** [comment]
Hi, I have the same problem. I described it on discord https://discord.com/channels/1098331563827134547/1296590313116143696
![image](https://github.com/user-attachments/assets/9edb6f97-9508-4ffe-962d-bbe483627ad0)
